docs/vgclog_examples.md

```diff
@@ -706,3 +707,895 @@ Analyzing the structure and elements of this example log output shows that this
+  <allocated-bytes non-tlh="0" tlh="0" arrayletleaf="0"/>
+</allocation-stats>
+<gc-op id="2065" type="global mark" timems="363.285" contextid="2062" timestamp="2021-03-05T12:16:44.639">
+  <trace-info objectcount="37481851" scancount="37457213" scanbytes="828974936" />
+  <finalization candidates="193" enqueued="0" />
+  <ownableSynchronizers candidates="2089" cleared="0" />
+  <references type="soft" candidates="3059" cleared="0" enqueued="0" dynamicThreshold="0" maxThreshold="32" />
+  <references type="weak" candidates="10797" cleared="0" enqueued="0" />
+  <references type="phantom" candidates="6" cleared="0" enqueued="0" />
+  <stringconstants candidates="10031" cleared="0"  />
+</gc-op>
+<gc-op id="2066" type="classunload" timems="0.122" contextid="2062" timestamp="2021-03-05T12:16:44.639">
+  <classunload-info classloadercandidates="25" classloadersunloaded="0" classesunloaded="0" anonymousclassesunloaded="0" quiescems="0.000" setupms="0.121" scanms="0.000" postms="0.000" />
+</gc-op>
+<gc-op id="2067" type="sweep" timems="5.080" contextid="2062" timestamp="2021-03-05T12:16:44.644" />
+<gc-op id="2068" type="compact" timems="800.059" contextid="2062" timestamp="2021-03-05T12:16:45.445">
+  <compact-info movecount="5788073" movebytes="120581384" />
+  <remembered-set-cleared processed="783167" cleared="783167" durationms="2.197" />
+</gc-op>
+<gc-end id="2069" type="global garbage collect" contextid="2062" durationms="1169.632" usertimems="7588.846" systemtimems="0.000" stalltimems="1403.135" timestamp="2021-03-05T12:16:45.445" activeThreads="8">
+  <mem-info id="2070" free="1572864" total="838860800" percent="0">
+    <mem type="eden" free="1572864" total="1572864" percent="100" />
+    <remembered-set count="879104" freebytes="29865984" totalbytes="33382400" percent="89" regionsoverflowed="0" regionsstable="0" regionsrebuilding="0"/>
+  </mem-info>
+</gc-end>
+<cycle-end id="2071" type="global garbage collect" contextid="2062" timestamp="2021-03-05T12:16:45.445" />
+<allocation-satisfied id="2072" threadId="00000000008C1F00" bytesRequested="24" />
+<af-end id="2073" timestamp="2021-03-05T12:16:45.445" threadId="00000000008C2880" success="true" />
+<exclusive-end id="2074" timestamp="2021-03-05T12:16:45.445" durationms="1177.699" />
+```
+-->
+
+#### Summary of the 'balanced' global GC cycle examples
+
+Analyzing the structure and elements of this example log output shows that this example global cycle has the following characteristics:
+
+- The global GC cycle was triggered during a global mark GC cycle.
+
+- The concurrent subincrement of the global mark GC cycle was interrupted by an allocation failure that triggered the concurrent subincrement to end and the `global mark` cycle type to change to a `global` type. 
+
+- The global GC cycle consists of only 1 GC increment, which runs mark, sweep and compact operations during an STW pause. 
+
+- The global GC cycle reclaimed the entire eden space of the heap.
```

Comment:
more impotant conculussion was that we ended up here because heap was very low in free memory (PGCs and GMPs could not recover memory), so it's expected to see very low free memory even after global GC. The application can recover from there, if it changes behavior (reduces load and amount of live objects).

Reply:
Changed to:

_Analyzing the structure and elements of this example log output shows that this example global cycle has the following characteristics:

- The global GC cycle was triggered during a global mark GC cycle when the heap was very low in memory. The memory could not be reclaimed by just using partial GC cycles and global mark cycles.

- The concurrent subincrement of the global mark GC cycle was interrupted by an allocation failure that triggered the concurrent subincrement to end and the `global mark` cycle type to change to a `global` type. 

- The global GC cycle consists of only 1 GC increment, which runs mark, sweep and compact operations during an STW pause. 

- The global GC cycle reclaimed the eden space, which is equivalently 1.5MB of memory. The heap initiates with 2048MB of free memory in the eden space, but [the amount of memory available in the eden space decreases during partial GC cycles](gc.md#balanced-policy), and in cases of tight memory conditions, reduces to a very low value._